artifact.go

```diff
@@ -18,7 +18,7 @@ func (a *artifactImpl) PathAfterFinished() (string, error) {
 		return "", errors.New("Path is not available when connecting remotely. Use SaveAs() to save a local copy")
 	}
 	path, err := a.channel.Send("pathAfterFinished")
-	if path == nil {
+	if path == nil { // < v1.40.0
```

Comment:
I personally would remove that, since we from the last person throw if you connect to a server which is not matching the protocol version you are using. The user use-case if that you are using the normal driver which we download for you where we ensure that the version always matches.